dll/win32/shell32/systray.cpp

```diff
@@ -50,11 +50,13 @@ BOOL WINAPI Shell_NotifyIconA(DWORD dwMessage, PNOTIFYICONDATAA pnid)
         nidW.cbSize = sizeof(nidW);
         dwValidFlags |= NIF_STATE | NIF_INFO | NIF_GUID /* | NIF_REALTIME | NIF_SHOWTIP */;
     }
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Comment:
Your intent in this file is good, but your changes are not correct:
V3 is XP-S2003 which is the wanted code, current sizeof() is Vista+ code...

Reply:
How to fix?

Reply:
V3 is Vista+.

Reply:
Do you agree Serge?

Reply:
> V3 is Vista+.

No, see #5730.

Reply:
`offsetof` is exclusive.
`FIELD_OFFSET` is inclusive.
You are misreading.

Reply:
> `FIELD_OFFSET` is upper-bound-inclusive. You are misreading.

I don't think so.

Reply:
`offsetof` and `FIELD_OFFSET` do the same thing (offset of the start of the member, i.e. exclusive). `RTL_SIZEOF_THROUGH_FIELD` includes the size of the member.

Reply:
Sorry, I was wrong. I apologize for my wrong saying.

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/8fd2decf-500f-4356-ab9e-504d8d25a20a)

Reply:
![image](https://github.com/reactos/reactos/assets/2107452/9db13dc7-729f-4eb5-a21a-d46fe392b536)

Reply:
> I apologize for my wrong saying.

No problem.

--

fix 2, big fix and big fix 2 are wrong, please revert them.
Especially, do not touch PSDK, which is correct as is.

Reply:
> How to fix?

Man. This PR is going crazier everyday.
You do need to sort out Vista code from systray.cpp (and elsewhere):
especially, V3_SIZE should not be used when compiling as NT5.2.